db/models.go

```diff
@@ -0,0 +1,57 @@
+package db
+
+import (
+	"time"
+
+	"gorm.io/gorm"
+)
+
+type Player struct {
+	gorm.Model
+	Name string `gorm:"unique;not null"`
+
+	PlayScores []PlayScore
+}
+
+type Play struct {
+	gorm.Model
+	Place string
+	Date  time.Time
+
+	PlayScores []PlayScore
+}
+
+// TODO{mikitachab} add wonders type enum
+type SevenWondersScore struct {
+	gorm.Model
+	WonderName           string
+	MilitaryConflicts    int
+	Wonders              int
+	TreasuryContents     int
+	CivilianStructures   int
+	ScientificStructures int
+	CommercialStructures int
+	Guilds               int
+
+	PlayScores []PlayScore
+}
+
+func (sws *SevenWondersScore) ScoreSum() int {
+	return sws.CivilianStructures +
+		sws.ScientificStructures +
+		sws.MilitaryConflicts +
+		sws.Wonders +
+		sws.TreasuryContents +
+		sws.CommercialStructures +
+		sws.Guilds
+}
+
+type PlayScore struct {
+	gorm.Model
+	PlayerID            uint
```

Comment:
Should we go with uint or int? As sqlite uses signed integers, maybe we should use int for IDs to be coherent with the database.

Reply:
> coherent with the database

whole point using ORM is to do not care what happening at DB level
>sqlite uses signed integers

and do not stick to specific database. sqlite just for dev. We want to use PosgreSQL in prod.